.github/workflows/test-unit.yaml

```diff
@@ -36,7 +36,23 @@ jobs:
           FUNC_REPO_BRANCH_REF: ${{ github.head_ref }}
       - name: Template Unit Tests
         run: make test-templates
+      - name: "Archive code coverage results"
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage-${{ matrix.os }}
+          path: ./coverage.txt
+          retention-days: 7
```

Comment:
It seems a bit long. I'd go with hours, e.g. 8h given it's per PR artifact and might iterate quickly.

Reply:
Would 7 days hurt anything?

Reply:
Also minimum is probably 1 day.

Reply:
not sure who pays for storage :)

I'm not really, it's very small file. But 1 day is plenty.

Reply:
>not sure who pays for storage :)

Uncle Bill?